encryption/hqc/hqc-rmrs/impl/ref.yaml

```diff
@@ -0,0 +1,5 @@
+name: Reference
+platform: 'Intel Core i7-7820X CPU, 3.6GHz '
+type: reference
+hardware features:
+  - 'Hyper-Threading, Turbo Boost and SpeedStep features were disabled '
```

Comment:
this is about which hardware features are required (for example, hardware aes acceleration or vector instructions). probably `[]` for the reference impl.